Deployments/suex-enginfprojects-acpt/k8s-manifests/autoscale-app.yml

```
@@ -0,0 +1,13 @@
+apiVersion: autoscaling/v1
```

```suggestion
apiVersion: autoscaling/v2
```
v1 is outdated. autoscaling/v2 API version, available since v1.23.

Ref: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v126